test/test_foreach.py

```diff
@@ -556,7 +566,10 @@ def test_binary_op_scalar_with_different_tensor_dtypes(self, device, dtype, op):
         self.assertIsNone(runtime_error)
 
     @skipIfTorchDynamo("Different error msgs, TODO")
-    @ops(foreach_binary_op_db, dtypes=OpDTypes.supported)
+    @ops(
+        filter(lambda op: op.name != "_foreach_clamp", foreach_binary_op_db),
+        dtypes=OpDTypes.supported,
+    )
```

Comment:
I think it would be valuable to add a similar checks for clamp if it's given an empty list: `with self.assertRaisesRegex(RuntimeError, "There were no tensor arguments to this function"):`. The other checks are definitely 2 lists specific.